test/e2e/resources/opc-pipeline.yaml

```diff
@@ -26,6 +26,8 @@ spec:
         - name: kubeconfig_dir
           workspace: kubeconfig_dir
       params:
+        - name: OPC_IMAGE
```

Comment:
you did not add under params here https://github.com/openshift-pipelines/task-openshift/pull/39/files#diff-797d622df4ce3c6f712499b05d88488c720105ff587044fce8aedda82d9f0aa0R14 then how did it ran :thinking: